drag_and_drop_v2/public/js/drag_and_drop.js

```diff
@@ -1115,6 +1115,7 @@ function DragAndDropBlock(runtime, element, configuration) {
 
             if (attemptsRemain()) {
                 data.misplaced_items.forEach(function(misplaced_item_id) {
+                    state.last_action_correct = false
```

Comment:
Okay, looks like #96 did not adjust the logic that sets the `correct` value on the response. It's not strictly in scope for this task, but I can add it if we want - @itsjeyd, @mtyaka, thoughts?